src/pages/EmergencyViewPage/EmergencyViewPage.tsx

```diff
@@ -0,0 +1,281 @@
+              <img
+                className="host-icon"
+                src={!venue.host?.icon ? DEFAULT_VENUE_LOGO : venue.host?.icon}
+                alt="host"
+              />
+            </div>
+
+            <div className="title">{venue.name}</div>
+
+            <div className="subtitle">
+              {venue.config?.landingPageConfig.subtitle}
+            </div>
+          </div>
+        </div>
+
+        <div className="row">
+          <div className="col-lg-6 col-12 venue-presentation">
+            <div>
+              <div
+                style={{ whiteSpace: "pre-wrap", overflowWrap: "break-word" }}
+              >
+                {venue.config?.landingPageConfig.description}
+              </div>
+
+              <div>
+                {venue.config?.landingPageConfig?.checkList?.map(
+                  (checkListItem: string, index: number) => (
+                    <div
+                      key={`checklist-item-${index}`}
+                      className="checklist-item"
+                    >
+                      <div className="check-icon-container">
+                        <FontAwesomeIcon icon={faCheckCircle} />
+                      </div>
+                      <div>{checkListItem}</div>
+                    </div>
+                  )
+                )}
+              </div>
+            </div>
+
+            {venue.config?.landingPageConfig.iframeUrl && (
+              <iframe
+                title="entrance video"
+                width="100%"
+                height="300"
+                className="youtube-video"
+                src={venue.config?.landingPageConfig.iframeUrl}
+                frameBorder="0"
+                allow={IFRAME_ALLOW}
+              />
+            )}
+
+            {venue.config?.landingPageConfig?.quotations?.map(
+              (quotation, index) => (
+                <div
+                  className="quotation-container"
+                  key={`${quotation.text}-${index}`}
+                >
+                  <div className="quotation">{quotation.text}</div>
+                  <div className="quotation-author">- {quotation.author}</div>
+                </div>
+              )
+            )}
+
+            {venue.config?.landingPageConfig?.presentation?.map(
+              (paragraph: string, index: number) => (
+                <p
+                  key={`venue-presentation-paragraph-${index}`}
+                  className="presentation-paragraph"
+                >
+                  {paragraph}
+                </p>
+              )
+            )}
+          </div>
+        </div>
+      </div>
+      <div className="EmergencyView">
+        <div className="EmergencyView_tabs">
+          {tabs.map((tab) => (
+            <div
+              className={`EmergencyView_tabs_item${
+                tab.id === selectedTab ? "--active" : ""
+              }`}
+              key={tab.id}
+              onClick={() => updateTab(tab.id)}
```

Comment:
`useCallback`. But better extracted into a separate component first not to pollute this one.